packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -355,14 +355,6 @@ class CpuProfileData {
     );
   }
 
-  /// Generate a cpu profile from [originalData] where each sample contains the
-  /// vmTag [tag].
-  ///
-  /// [originalData] does not need to be [processed] to run this operation.
-  factory CpuProfileData.fromVMTag(CpuProfileData originalData, String tag) {
```

Comment:
@bkonyi is this a bug that this code is unused?

Reply:
Same as https://github.com/flutter/devtools/pull/5949#discussion_r1342165085

Reply:
@bkonyi please advise

Reply:
Waiting for this one and then will address all latest comments and bump the version.

Reply:
Sorry for the delay on this.

I think it can be removed, it was probably added for symmetry and then never used.